src/main/java/com/datatorrent/netlet/DefaultEventLoop.java

```diff
@@ -513,14 +512,18 @@ public String toString()
 
             });
           }
-        }
-        catch (IOException ie) {
-          l.handleException(ie, DefaultEventLoop.this);
+        } catch (UnresolvedAddressException e) {
+          logger.error("Inet Address {} is not resolvable.", address, e);
```

Comment:
My personal preference is to log all exception in `AbstractClient.handleException()` using error level. This frees library consumers from overwriting `handleException()` only to log the exception and in most cases it is the only action that the netlet consumer can take as `handleException()` is called on the `defaultEventLoop` thread. In cases where logging is not desired, it can be disabled by either overwriting "handleException' or by adjusting "com.datatorrent.netlet" logger level in log4j.

Many libraries, including Hadoop client library log errors and warnings, for example Hadoop will log a warning if it can't find or load native libraries.

If the concern is that the exception may be logged twice, it is a less severe issue than not logging or reporting error at all.

If everyone agrees on that approach, I will redo the commit.